cloudlaunchserver/values.yaml

```diff
@@ -89,7 +89,8 @@ affinity: {}
 
 rabbitmq:
   rabbitmqUsername: notAGuest
-  rabbitmqPassword: aRandomPassword
+  # Will autogenerate if not set
+  # rabbitmqPassword: aRandomPassword
   rabbitmqErlangCookie: mustRemainSameBetweenUpgrades
```

Comment:
ErlangCookie also needs to be generated and stored securely in a secret, but need not be settable through values.

Reply:
Should it have no default and be required?

Reply:
(at that point I would say we can auto-generate a value in `cloudman-boot` for the GVL)

Reply:
Or should we instead have a helm install hook? I believe if we don't set it the randomly generated value will change during upgrades

Reply:
It looks like simply not defining it should do the trick?
https://github.com/helm/charts/blob/2ea77442d176f194b82b38fed7d5fa0fc9470883/stable/rabbitmq-ha/templates/secret.yaml#L25

Reply:
Have added this while merging this commit.